src/Jackalope/Node.php

```diff
@@ -1462,17 +1478,6 @@ public function confirmSaved()
     }
 
     /**
-     * In addition to calling parent method, tell all properties
-     */
-    public function setPath($path, $move = false)
```

Comment:
assume this:

```php
$n = $session->getNode('/bar');
$p = $n->getProperty('foo');
$session->move('/bar', '/other');
echo $p->getPath();
```

we do still need setPath, but only update the properties that are already loaded